app/certificates.go

```diff
@@ -404,39 +430,73 @@ func NewCertificatesCommand() (CommandOut, error) {
 							CaPrivateKeyPEM: caPrivKey,
 						})
 						if err != nil {
-							return fmt.Errorf("failed to generate certificate: %s", err)
+							return fmt.Errorf("failed to generate certificate: %w", err)
 						}
-						yes, err := ConfirmPrompt(
+						return writeCertificates(
 							ctx,
-							fmt.Sprintf("storing the certificate private key at %s, do not share this key with anyone. confirm:",
-								ctx.Path("private-key-file"),
-							),
-						)
-						if err != nil || !yes {
-							return nil
-						}
-						err = ioutil.WriteFile(
-							ctx.Path("certificate-file"),
+							"client certificate",
 							certPem,
-							0600,
-						)
-						if err != nil {
-							return fmt.Errorf("failed to write ca certificate: %s", err)
-
-						}
-						err = ioutil.WriteFile(
-							ctx.Path("private-key-file"),
 							certPrivKey,
-							0600,
+							ctx.Path("certificate-file"),
+							ctx.Path("key-file"),
 						)
-						if err != nil {
-							return fmt.Errorf("failed to write ca private key: %s", err)
-						}
-
-						return nil
 					},
 				},
 			},
 		},
 	}, nil
 }
+
+func checkPath(ctx *cli.Context, path string) (bool, error) {
+	if fi, err := os.Stat(path); !errors.Is(err, os.ErrNotExist) {
```

Comment:
this is assuming that an error returned other than "not found" assumes the file exists? but that may also not be true?